amgut/templates/secondary_survey.html

```diff
@@ -0,0 +1,84 @@
+{% extends sitebase.html %}
+{% from wtforms import validators %}
+{% block head %}
+<import >
+<script type="text/javascript">
+    $(document).ready(function() {
+      $("div[id^=supp_]").hide();
+    });
+</script>
+<style>
+input[type="checkbox"] {
```

Comment:
Are this style modifications specific for this page or are general to all surveys (i.e. are happening in all surveys). If the former, why? If the later, why they've not been done in a single CSS and include it, minimizing code duplication?